packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Inbound/XCA ITI-38 Inbound Interface.xml

```diff
@@ -5716,8 +5716,8 @@ Last updated: Jan 02 2024
         <timezone>America/Los_Angeles</timezone>
       </lastModified>
       <pruningSettings>
-        <pruneMetaDataDays>5</pruneMetaDataDays>
-        <pruneContentDays>5</pruneContentDays>
+        <pruneMetaDataDays>15</pruneMetaDataDays>
+        <pruneContentDays>15</pruneContentDays>
```

Comment:
Increasing the data retention for inbound just in case the pruner gets enabled there so we don't lose data while debugging it.

Reply:
But will this increase memory usage?

Reply:
It shouldn't, the Pruner is not running on inbound right?